src/format/dashboard/format-dashboard.ts

```diff
@@ -110,7 +110,10 @@ export function dashboardFormat() {
           const formats: Record<string, Metadata> = format.metadata
             .format as Record<string, Metadata>;
           const htmlFormat = formats["html"];
-          if (htmlFormat && htmlFormat[kTheme]) {
+          const dashboardFormat = formats["dashboard"];
+          if (dashboardFormat && dashboardFormat[kTheme]) {
+            format.metadata[kTheme] = dashboardFormat[kTheme];
+          } else if (htmlFormat && htmlFormat[kTheme]) {
```

Comment:
> Is this change meant to pick up any dashboard theme format defined at project level ?

This relates to the question I asked on Slack yesterday:

- Before this PR, an `html` theme would be picked up by the `dashboard` format and used.
- With this PR and without this specific change, themes need to be set to `dashboard` specifically, even if they were set in the `html` format. This is a change from the 1.5 behavior
- With this PR and with this specific change, a theme set to `html` will affect `dashboard`, mimicking the 1.5 behavior. Changing the theme specifically in `dashboard` works, and overrides the theme for dashboards in case it exists for HTML.

Reply:
Thanks a lot for the clear explanation. That makes sense - I see now the specific of third bullet point. 

IMO we need to highlight this change and the impact. But it definitely makes sense to have this behavior now. Thank you